metrics/web/docs/features/public_dashboard/diagrams/public_dashboard_sequence_diagram.puml

```diff
@@ -0,0 +1,60 @@
+@startuml public_dashboard_sequence_diagram
+
+skinparam ParticipantPadding 20
+skinparam BoxPadding 10
+
+actor Bob
+
+participant Browser
+participant LoadingPage
+participant AuthNotifier
+participant FeatureConfigNotifier
+participant NavigationNotifier
+participant CoolPage
+participant LoginPage
+
+Bob -> Browser ++:  opens a "Cool Page" URL
+Browser -> LoadingPage ++: navigates to initialize all components
+LoadingPage --> Browser: returns the LoadingPage UI
+Browser --> Bob: sees the LoadingPage UI
+
+LoadingPage -> AuthNotifier ++: addListener(_authNotifierListener)
+AuthNotifier --> LoadingPage --: the listener is added
```

Comment:
🔨 
I think this arrow is redundant since `addListener` is a `void` method and there is no response from the `AuthNotifier` to the `LoadingPage` about the listener is added.